src/plugins/remote-cache/routes/get-status.ts

```diff
@@ -0,0 +1,21 @@
+import type { Server } from 'http'
+import type { RawReplyDefaultExpression, RawRequestDefaultExpression, RouteOptions } from 'fastify'
+import { type Params, type Querystring } from './schema'
+import { statusRouteSchema } from './status-schema'
+
+export const getStatus: RouteOptions<
+  Server,
+  RawRequestDefaultExpression,
+  RawReplyDefaultExpression,
+  {
+    Querystring: Querystring
+    Params: Params
+  }
+> = {
+  method: 'GET',
+  url: '/artifacts/status',
+  schema: statusRouteSchema,
```

Comment:
It is better not to log every time this route is hit since it can be hit several times (when used for liveness probes).
Add
```suggestion
  schema: statusRouteSchema,
  logLevel: 'silent'
```